cli/reporter/print_compact.go

```diff
@@ -373,20 +373,22 @@ func (r *defaultReporter) printAssetQueries(resolved *policy.ResolvedPolicy, rep
 		r.out.Write([]byte(NewLineCharacter))
 	}
 
-	r.out.Write([]byte("Controls:" + NewLineCharacter))
-	for id, score := range report.Scores {
-		_, ok := resolved.CollectorJob.ReportingQueries[id]
-		if !ok {
-			continue
-		}
+	if len(report.Scores) >= 0 {
```

Comment:
was this meant to be `>0` instead of >= ? else the check does not make a lot of sense